import_food.py

```diff
@@ -261,11 +255,7 @@ def remove_some_processes(db):
     )
     args = parser.parse_args()
 
-    projects.set_current(PROJECT)
-    # projects.create_project(PROJECT, activate=True, exist_ok=True)
```

Comment:
this comment was due to an intermediate and reverted change on the project managements on the bw side ;-)